docs/tutorial/01-mlrun-basics.ipynb

```diff
@@ -334,15 +334,15 @@
    "source": [
```

Comment:
<p>... within your Jupyter pod. This means that the&nbsp;...</p><p> </p><ul><li><code>name</code>—the job name</li><li><code>handler</code>—the name of the function handler</li><li><code>inputs</code>—the data-set URL</li></ul><p>The example uses a CSV file from a cloud object-store service named wasabisys as input for the function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>